andela/mont-blanc-rc
# #152957719 configure circleci

### What does this PR do?
+ Configure circleci for codebase.

### Description of Task to be completed?
CircleCi checks if tests for codebase is properly written;
+ Create an account on circleci and add github project repo.
+ Add circleci badge.

### How should this be manually tested?
+ Raise PR request and check if build passes or fails; this is indicated on the circleci badge.

### Any background context you want to provide?
+ None yet

### What are the relevant pivotal tracker stories?
`#152957719`

### Screenshots
![image](https://user-images.githubusercontent.com/23665997/33028908-6ef6581e-ce17-11e7-995e-7f263aa8ad21.png)


### Questions:
+ None yet
